src/test/spec/unified_runner/operation.rs

```diff
@@ -1556,12 +1559,14 @@ impl TestOperation for CreateCollection {
                         .await
                 })
                 .await?;
+                collection = Entity::Collection(database.collection(&self.collection));
```

Comment:
It looks like this is the same in both `if` branches - could the assignment be moved to just after the if?